ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -38,16 +37,16 @@ android {
 }
 
 dependencies {
-    implementation('com.vungle:publisher-sdk-android:6.3.24') {
-        transitive=true
```

Comment:
not sure if omitting transitive would affect anything on AdMob side, also I thought that its implied,
was it done initially on older repo/branch and got carried over as-is

Reply:
Done. Add transitive back.